### ISSUE TITLE
`SlackApprovalAction` fails due to outdated node runtime in Lambda function

### ISSUE THREAD

**Issue by calvinwyoung** [cloudcomponents/cdk-constructs#143]
I'm trying to use the `SlackApprovalAction` from `@cloudcomponents/cdk-codepipeline-slack`, but I receive the following error when I run it:

```
Resource handler returned message: "The runtime parameter of nodejs10.x is no longer supported for 
creating or updating AWS Lambda functions. We recommend you use the new runtime (nodejs14.x) while
creating or updating functions.
```

Is there any reason that lambda needs to use nodejs10.x? None of the lambdas are doing anything particularly complex so it *appears* to be safe to upgrade.

**Comment by hupe1980** [comment]
There is no reason for that. It's just a mistake.

**Comment by calvinwyoung** [comment]
Awesome thanks for the quick response — I opened a small PR to fix this in #144.

**Comment by hupe1980** [comment]
Closed with #144
